config/routes.rb

```diff
@@ -1,6 +1,16 @@
 Rails.application.routes.draw do
+  # Define your application routes per the DSL in https://guides.rubyonrails.org/routing.html
+
+  # Defines the root path route ("/")
+  # root "articles#index"
   root 'users#index'
-  resources :users, only: %i[index show] do
-    resources :posts, only: %i[index show]
-  end
+  get '/users', to: 'users#index', as: :users
+  get '/users:id', to: 'users#show', as: :user
+  get 'users:user_id/posts', to: 'posts#index', as: :user_posts
+  get 'users:user_id/posts:id', to: 'posts#show', as: :user_post
+  get '/posts/new', to: 'posts#new', as: :new_user_post
+  post '/posts/new', to: 'posts#create' #create_new_post
+  post 'comments/new', to: 'comments#create', as: :new_user_comment #create_new_comment
+  post 'likes/new', to: 'likes#create', as: :new_user_like #create_new_like
+  get 'users/:user_id/post/:id' => 'comments#new', as: 'user_comment'
```

Comment:
- [ ] [OPTIONAL] Please modify your routes as you are currently not applying best practices to your routing. For instance,
Instead of 
```ruby 
get '/users:id', to: 'users#show', as: :user
```
You should have
```ruby 
get '/users/:id', to: 'users#show', as: :user
```
**Resource:** Learn more about RESTFUL routing [here](https://www.learnhowtoprogram.com/c-and-net/basic-web-applications/introduction-to-restful-routing)